docs/gettingstarted.rst

```diff
@@ -0,0 +1,110 @@
++-------------+--------------------------------------------+----------------------------------------------------------------------------------------------------+
+| choices?    | array of `ApplicationCommandOptionChoice`_ | choices for ``string`` and ``int`` types for the user to pick from                                 |
++-------------+--------------------------------------------+----------------------------------------------------------------------------------------------------+
+| options?    | array of `ApplicationCommandOption`_       | if the option is a subcommand or subcommand group type, this nested options will be the parameters |
++-------------+--------------------------------------------+----------------------------------------------------------------------------------------------------+
+
+This table shows us the way that Discord handles the structure of options for
+slash commands through their Bot API. For visualization purposes, we'll quickly
+make a JSON example (although we won't be using it) in order to explain how this
+works:
+
+.. code-block:: python
+
+  {
+    "name": "argOne",
+    "description": "description of first argument",
+    "type": 3, # STRING type,
+    "required": True
+  }
+  
+With this very basic understanding in mind, now we are able to begin programming
+a simple Python script that will allow us to utilize this ability through one of
+the many subclasses offered in *discord-py-slash-command*.
+
+.. code-block:: python
+
+  import discord
+  from discord_slash import SlashCommand
+  from discord_slash.utils import manage_commands # Allows us to manage the command settings.
+
+  client = discord.Client(intents=discord.Intents.all())
+  slash = SlashCommand(client, auto_register=True)
+
+  guild_ids = [789032594456576001]
+
+  @client.event
+  async def on_ready():
+      print("Ready!")
+
+  @slash.slash(
+    name="test",
+    description="this returns the bot latency",
+    options=[manage_commands.create_option(
+      name = "argOne",
+      description = "description of first argument",
+      option_type = "STRING", # We can pass the name of the type directly
+      required = True
+    )],
+    guild_ids=guild_ids
+  )
+  async def _test(ctx, argOne: str):
+      await ctx.send(content=f"You responded with {argOne}.")
+
+  client.run("your_bot_token_here")
+  
+The main changes that you need to know about are with the lines calling the import
+of ``manage_commands``, as well as the ``options = [] ...`` code within the ``@slash.slash()``
+context coroutine. This will now create a new option called "argOne" when shown for
+the slash command.
+
+.. _quickstart: https://discord-py-slash-command.readthedocs.io/en/latest/quickstart.html
+.. _ApplicationCommandOptionChoice: https://discord.com/developers/docs/interactions/slash-commands#applicationcommandoptionchoice
+.. _ApplicationCommandOption: https://discord.com/developers/docs/interactions/slash-commands#applicationcommandoption
+
+.. toctree::
```

Comment:
We don't actually need toctree and `Indices and tables` for extra docs page.
It is only necessary for the index page.

Reply:
Ah okay, I will remove it then.